docs/extensions/httpfs.md

```diff
@@ -217,3 +217,15 @@ Some additional configuration options exist for the S3 upload, though the defaul
 | `s3_uploader_max_parts_per_file` | used for part size calculation, see [AWS docs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html) |
 | `s3_uploader_max_filesize` | used for part size calculation, see [AWS docs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/qfacts.html) |
 | `s3_uploader_thread_limit` | maximum number of uploader threads |
+
+Additionally, most of the configuration options can be set via environment variables:
+
+| DuckDB setting          | Environment variable    |
+|:------------------------|:------------------------|
+|  `s3_region`            | `AWS_REGION`            |
+|  `s3_region`            | `AWS_DEFAULT_REGION`    |
```

Comment:
Which one takes priority when both `AWS_DEFAULT_REGION` and `AWS_REGION` are set?

Reply:
AWS_REGION takes priority over AWS_DEFAULT_REGION, can add that as a note column